lib/slide_hub/convert_util.rb

```diff
@@ -1,4 +1,4 @@
-module Oss
+module SlideHub
   class ConvertUtil
```

Comment:
__[reek]__
* [DataClump] takes parameters [dir, file] to 4 methods [:ref](https://github.com/troessner/reek/blob/master/docs/Data-Clump.md)